### DISCUSSION TITLE
Privacy based analytics verified widget

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#107] [Anything Else]
"Would help display to site visitors that the owners care for privacy while in turn promoting plausible itself to people who might be interested in trying it for their own sites.

If you were to install this on your website, what content would you like the widget to have? Would you want to show off how many current users are on the website?

Not so much a popup, (that could be a private based alt to provesource lol) but like a little embed like iubenda does with its privacy policy like a button that takes people to an info page about plausible and shows that x site cares abt privacy on the widget. Would be a good way to expose your product too!"

([imported from the old roadmap](https://plausible.nolt.io/48))

**Comment by metmarkosaric** [comment]
"Add "I'm a part of the Plausible pledge", a pledge that people take for privacy that can be displayed to users of sites that use Plausible. Optional setting."

([imported from the old roadmap](https://plausible.nolt.io/57))

**Comment by kasbah** [comment]
I'm not too interested in a very wordy thing with "I'm part of a pledge" etc. but would add a nice badge with just visitor count to our readme on Github and an `/about` page on the site. Something like Simple Analytics' one: https://simpleanalytics.com/badges

**Comment by divinerites** [comment]
OK. So before being caught by M. Dunning & ­Kruger this is an attempt with 3 badges. A (really) flashy one, a sobber one and a classical one.

[EDIT] This will raise the difficult question about "is 0% a plural ?" 😸 
 
![privacy-1](https://user-images.githubusercontent.com/32342823/90277239-8a34ca00-de65-11ea-8dd5-8f86b506f1ee.png)

![privacy-2](https://user-images.githubusercontent.com/32342823/90277253-8f921480-de65-11ea-9508-41eb8b457fc9.png)

![privacy-3](https://user-images.githubusercontent.com/32342823/90277266-9456c880-de65-11ea-9790-b9ddcbce405f.png)
